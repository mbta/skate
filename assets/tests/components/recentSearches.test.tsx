import React from "react"
import renderer from "react-test-renderer"
import RecentSearches from "../../src/components/recentSearches"

describe("RecentSearches", () => {
  test("renders", () => {
    const tree = renderer.create(<RecentSearches />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
