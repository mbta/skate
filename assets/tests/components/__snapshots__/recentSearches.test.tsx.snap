// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentSearches renders 1`] = `
<div
  className="m-recent-searches"
>
  <div
    className="m-recent-searches__heading"
  >
    Recent Searches
  </div>
</div>
`;
